int64 timestamp
sensor_msgs/Image map